src/components/MoveControl.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
```

Comment:
Hi @mogavin, Could you please put the copyright holder and license inside this file?

Reply:
Done !